### PR TITLE
[INLONG-1831][Website] Add official website content search button

### PR DESCRIPTION
**Pull request by Chever-John**
Fixes https://github.com/apache/incubator-inlong/issues/1831

where *XYZ* should be replaced by the actual issue number.

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [x] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Comment by leezng** [comment]
@Chever-John 
Did you use your private email to register the appId and key on algolia?

File: docusaurus.config.js [hunk 1]
```diff
@@ -238,6 +238,12 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
         theme: require('prism-react-renderer/themes/dracula'),
         darkTheme: darkCodeTheme,
       },
+      algolia: {
+        appId: 'YUW9QEL53E',
```
Comment (leezng) [inline docusaurus.config.js:242]: @Chever-John
Did you use your private email to register the appId and key on algolia?

**Comment by dockerzhang** [comment]
@Chever-John the CI build failed, please rebase the master for https://github.com/apache/incubator-inlong-website/pull/388 merged.

**Comment by Chever-John** [comment]
@dockerzhang Sure, sir~

**Comment by Chever-John** [comment]
@dockerzhang Yes, I have successfully completed the rebase, but using the force push method. Should I be careful not to use force push for community commits?

File: docusaurus.config.js (hunk 1 above)
Comment (Chever-John) [reply docusaurus.config.js:242]: This is true as far as the facts are concerned. But I think I got access to Alex before I operated. I forwarded the email to Alex.

**Comment by dockerzhang** [comment]
> @dockerzhang Yes, I have successfully completed the rebase, but using the force push method. Should I be careful not to use force push for community commits?

it's a usual operation for `force push` after rebase.

File: docusaurus.config.js (hunk 1 above)
Comment (dockerzhang) [reply docusaurus.config.js:242]: Do we have an expiry date for the appId and key? if it works for a long time, I think the private email is not a big deal.

**Comment by Chever-John** [comment]
@dockerzhang This is a potential problem to consider. Let me check it and give you a reply.

**Comment by Chever-John** [comment]
@dockerzhang By the way, when I've solved problems in some communities, they don't recommend using rebase and force push code when the code has already been committed. So I have this question~

File: docusaurus.config.js (hunk 1 above)
Comment (Chever-John) [reply docusaurus.config.js:242]: Hi, I am back for the newest status.
1. I have consulted with the relevant developers at Apache APISIX and Apache Skywalking. Based on their experience, they believe that it is perfectly fine to register using a private account, and then if you need to replace it in the future, email the official.
2. I didn't get a valid answer about the key's expiration date, and I'm waiting for an update. It seems that everyone doesn't care about it.
3. I am waiting for the official's reply.

**Review by dockerzhang — APPROVED** [review]

**Comment by dockerzhang** [comment]
@Chever-John LGTM, thanks.

**Comment by Chever-John** [comment]
@dockerzhang My pleasure.

**Review by bluewang — APPROVED** [review]

**Review by leezng — APPROVED** [review]

**Comment by Chever-John** [comment]
@dockerzhang @bluewang @leezng I found the search engine seems do not to work. Let me check it.

**Comment by bluewang** [comment]
> @dockerzhang @bluewang @leezng I found the search engine seems do not to work. Let me check it.

Is there any progress on this situation?

**Comment by Chever-John** [comment]
@bluewang Give me a few days, I'll give it a try.

**Comment by bluewang** [comment]
> @bluewang Give me a few days, I'll give it a try.

@Chever-John OK. Waiting for your good news

**Pull request merged**